<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightDlg</class>
 <widget class="QDialog" name="FlightDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>飞机型号</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string> 飞机型号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="HuacaoManagement.qrc">
     <normaloff>:/HuacaoManagement/Resources/1197037.png</normaloff>:/HuacaoManagement/Resources/1197037.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="HuacaoManagement.qrc">
     <normaloff>:/HuacaoManagement/Resources/1197030.png</normaloff>:/HuacaoManagement/Resources/1197030.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidgetFlight">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>210</width>
     <height>140</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">航班号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>座位数</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>208</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HuacaoManagement.qrc"/>
 </resources>
 <connections/>
</ui>
